packages/fxa-customs-server/lib/config/config.js

```diff
@@ -43,11 +43,25 @@ module.exports = function(fs, path, url, convict) {
     },
     limits: {
       blockIntervalSeconds: {
-        doc: 'Duration of a manual ban',
+        doc: 'Duration of a manual `block` ban',
         default: 60 * 60 * 24,
         format: 'nat',
         env: 'BLOCK_INTERVAL_SECONDS',
       },
+      suspectInterval: {
+        doc:
+          'Duration of a `suspect` flag, instigated by the Dataflow pipeline',
+        default: '1 day',
+        env: 'SUSPECT_INTERVAL',
+        format: 'duration',
+      },
+      disableInterval: {
+        doc:
+          'Duration of a long-term `disable` flag, instigated by the Dataflow pipeline',
+        default: '1 year',
+        env: 'DISABLE_INTERVAL',
+        format: 'duration',
```

Comment:
Are durations automatically converted to MS? Should the `nat`s be converted to use `duration` (not for this PR of course)?

Reply:
> Are durations automatically converted to MS?

Yep, see:

https://github.com/mozilla/node-convict/blob/7483615640c4cfc0c9466744ec85a86a6460edcb/test/format-tests.js#L64-L67

And then:

https://github.com/mozilla/node-convict/blob/7483615640c4cfc0c9466744ec85a86a6460edcb/test/format-tests.js#L180-L182

> Should the `nat`s be converted to use `duration` (not for this PR of course)?

I considered the same, but am somewhat wary of making broad changes in this repo. But in theory, yep, they should be durations too.